signac/contrib/project.py

```diff
@@ -834,13 +834,13 @@ def _register(self, _id, statepoint):
 
         Parameters
         ----------
-        _id : str
+        id_ : str
```

Comment:
the actual id argument of the `_register` method was not changed here

Reply:
@lyrivera Just to clarify, would you like _id in the function signature to be changed to id_?

Reply:
Thank you very much - this has been sorted :)